src/api.js

```diff
@@ -77,3 +77,24 @@ export const logoutUser = async token => {
     throw error;
   }
 };
+
+export const getFavouritedConferenceById = async (id, token) => {
```

Comment:
Could be nice to pluralise this to `getFavouritedConferencesById` or even `byUserId` to be more obvious what its doing